src/init.cpp

```diff
@@ -899,9 +899,9 @@ bool AppInit2(boost::thread_group& threadGroup, CScheduler& scheduler)
     try {
         static boost::interprocess::file_lock lock(pathLockFile.string().c_str());
         if (!lock.try_lock())
-            return InitError(strprintf(_("Cannot obtain a lock on data directory %s. Bitcoin Core is probably already running."), strDataDir));
+            return InitError(strprintf(_("Cannot obtain a lock on data directory %s. Bitcoin Classic is probably already running."), strDataDir));
```

Comment:
Does Classic use a different lock from Core? Any Bitcoin client might be running, I supposed?